prime-router/src/main/kotlin/azure/HistoryFunctions.kt

```diff
@@ -175,13 +176,31 @@ class GetReports :
     fun run(
         @HttpTrigger(
             name = "getReports",
-            methods = [HttpMethod.GET],
+            methods = [HttpMethod.GET, HttpMethod.HEAD, HttpMethod.OPTIONS],
```

Comment:
Why the addition of `HEAD` and `OPTIONS even though `getReports` doesn't differentiate between these methods?